library/src/main/java/com/devbrackets/android/exomedia/ExoMedia.java

```diff
@@ -37,6 +37,8 @@
         METADATA
     }
 
+    public static final int TRACK_GROUP_DISABLED  = -1;
```

Comment:
This doesn't belong here, at the very least it belongs in the ExoMediaPlayer class